pulp_container/app/viewsets.py

```diff
@@ -100,6 +102,14 @@ class Meta:
         fields = BaseDistributionViewSet.filterset_class.Meta.fields
 
 
+class ContainerNamespaceFilter(BaseFilterSet):
```

Comment:
I had to add it to be able to search for namespaces to  clean them up in tests.
Let me know if it's worth a dedicated issue/log entry/pr and I can separate it from this never-ending-solve-all-the-other-issues-around PR.

Reply:
It's fine for me. We should be able to assume there are filters for every viewset...